examples/poker/client/web/src/components/ActivePot.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
do you need this import even if you're not using it explicitly in this file? vscode tells me it's an unused import

Reply:
yeah we don't need it anymore with react 18, I am still used to importing everywhere. will be removing